src/components/FeatureViews/Onboarding/ImportFlow.js

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import {makeStyles} from '@material-ui/core/styles';
+import Stepper from '@material-ui/core/Stepper';
+import Step from '@material-ui/core/Step';
+import StepLabel from '@material-ui/core/StepLabel';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+
+import DownloadTemplate from "./DownloadTemplates";
+import CourseUpload from "./CourseUpload";
+import BusinessInfo from "./BusinessInfo";
+import CategorySelect from "./CategorySelect";
+import AccountsUpload from "./AccountsUpload";
+
+const useStyles = makeStyles((theme) => ({
+	root: {
+		width: '100%',
+	},
+	button: {
+		margin: "0 35px"
+	},
+	instructions: {
+		marginTop: theme.spacing(1),
+		marginBottom: theme.spacing(1),
+	},
+}));
+
+function getSteps() {
+	return [
+		'Business info',
+		'Templates',
+		'Course Categories',
+		'Accounts',
+		'Courses'
+	];
+}
+
+function getStepContent(step) {
+	switch (step) {
+		case 0:
+			return <BusinessInfo/>;
+		case 1:
+			return <DownloadTemplate/>;
+		case 2:
+			return <CategorySelect/>;
+		case 3:
+			return <AccountsUpload/>;
+		case 4:
+			return <CourseUpload/>;
+		default:
+			return 'Unknown step';
```

Comment:
I feel like the user would get confused if "Unknown step" was displayed, could we instead throw some kind of error or be more descriptive?